modules/admin/src/resources/js/directives.js

```diff
@@ -659,6 +659,7 @@
             scope: {
                 "model": "=",
                 "options": "=",
+                "dateFormat": "=",
```

Comment:
I assume this wont work:

> Note: These =attr attributes in the scope option of directives are normalized just like directive names. To bind to the attribute in <div bind-to-this="thing">, you'd specify a binding of =bindToThis.

From the angular docs (https://docs.angularjs.org/guide/directive)